packages/migrate/migrations/app-state/index.js

```diff
@@ -94,8 +95,10 @@ export async function migrate() {
 	/** @type {(s: string) => string} */
 	const cyan = (s) => pc.bold(pc.cyan(s));
 
-	// TODO: use package-manager-detector here
-	const tasks = ["install the updated dependencies ('npm i' / 'pnpm i' / etc)"];
+	const detected = await detect({ cwd: process.cwd() });
+	const pm = detected?.name ?? 'npm';
+
+	const tasks = [`Install the updated dependencies by running ${cyan(`${pm} install`)}`];
```

Comment:
```suggestion
	const detected = await detect({ cwd: process.cwd() });
	const pm = detected?.name ?? 'npm';
	const cmd = /** @type {import('package-manager-detector').ResolvedCommand} */ (
		resolveCommand(pm, 'install', [])
	);

	const tasks = [
		`Install the updated dependencies by running ${cyan(`${cmd.command} ${cmd.args.join(' ')}`)}`
	];
```

Hardcoding the `install` command might not be correct in each case. I think for the install command it should work basically everywhere, but let's stay agnostic if that abstraction already exists.

Same would go for the other places.